src/devices/CharacterLcd/README.md

```diff
@@ -17,6 +19,22 @@ using (var lcd = new Lcd1602(18, 5, new int[]{6, 16, 20, 21})) //using 4 data pi
 }
 ```
 
+Grove LCD RGB Backlight uses two i2c devices:
+- the device to control LCD (address 0x3E)
+- the device to control RGB backlight (address 0x62)
+
+Here is a Hello World example of how to consume Grove LCD RGB Backlight binding:
+```c#
+var i2cLcdDevice = new UnixI2cDevice(new I2cConnectionSettings(busId: 1, deviceAddress:0x3E));
+var i2cRgbDevice = new UnixI2cDevice(new I2cConnectionSettings(busId: 1, deviceAddress:0x62));
+using (var lcd = new LcdRgb1602(i2cLcdDevice, i2cRgbDevice))
+{
+    lcd.Write("Hello World!");
+    lcd.SetColor(Color.Azure);
+}
+```
+
 ## References 
 - Very complete tutorial on how to connect and work with one of these displays: https://learn.adafruit.com/drive-a-16x2-lcd-directly-with-a-raspberry-pi/overview
 - Good guide explaining how the device works internally: http://www.site2241.net/november2014.htm
+- Seeedstudio, Grove - LCD RGB Backlight library: https://github.com/Seeed-Studio/Grove_LCD_RGB_Backlight
```

Comment:
might be worth adding some attribution in https://github.com/dotnet/iot/blob/master/THIRD-PARTY-NOTICES.TXT

Reply:
Sorry, I'm not sure how to fill this information properly... Could you help me with it, please?

Reply:
@SergeyRazmyslov here you can find some examples: https://github.com/dotnet/corefx/blob/master/THIRD-PARTY-NOTICES.TXT - this is not strictly required but it's good to attribute the person you based your code on

Reply:
Thank you @krwq  for clarification! I've added this info as part of #248 .